lighthouse-cli/test/fixtures/form.html

```diff
@@ -0,0 +1,128 @@
+<!doctype html>
+<!--
+ * Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+-->
+<html><head>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Autofill Smoke Test</title>
+</head>
+<body>
+  <form id="checkout" name="checkout" action="../done.html" method="post">
+      <label id="cardName" for="name_cc">Name on card:</label> <textarea type="text" id="name_cc" autocomplete="cc-name"></textarea><br>
+      <label id="cardNumber" for="CCNo">Credit card number:</label> <input type="text" id="CCNo" autocomplete="cc-number"><br>
+      <label id="expDate" for="CCExpiresMonth">Expiry Date:</label> <select id="CCExpiresMonth" autocomplete="cc-exp-month">
+        <option value="">MM</option>
+        <option value="01">01</option>
+        <option value="02">02</option>
+        <option value="03">03</option>
+        <option value="04">04</option>
+        <option value="05">05</option>
+        <option value="06">06</option>
+        <option value="07">07</option>
+        <option value="08">08</option>
+        <option value="09">09</option>
+        <option value="10">10</option>
+        <option value="11">11</option>
+        <option value="12">12</option>
+      </select> / <select id="CCExpiresYear" autocomplete="cc-exp-year">
+        <option value="">YY</option>
+        <option value="2019">2019</option>
+        <option value="2020">2020</option>
+        <option value="2021">2021</option>
+        <option value="2022">2022</option>
+        <option value="2023">2023</option>
+        <option value="2024">2024</option>
+        <option value="2025">2025</option>
+        <option value="2026">2026</option>
+        <option value="2027">2027</option>
+        <option value="2028">2028</option>
+        <option value="2029">2029</option>
+      </select><br>
+      <label id="cvc" for="cvc">CVC:</label> <input id="cvc" autocomplete="cc-csc"><br>
```

Comment:
as @connorjclark already pointed out there are a lot of duplicate `id`s in this test file, is there a reason all the labels have IDs to begin with? seems unnecessary and might just want to sprinkle in a few to ensure we can check the property

Reply:
@patrickhulce  yea I merged two forms and completely forgot about the id's being similar, implementing that change right now